resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/submit/KubernetesClientApplication.scala

```diff
@@ -180,6 +179,26 @@ private[spark] class Client(
       originalMetadata.setOwnerReferences(Collections.singletonList(driverPodOwnerReference))
     }
   }
+
+  // Build a Config Map that will house both the properties file and the java options file
```

Comment:
Maybe an incorrect javadoc, or else this is erroneously not putting in the java options file? Also where do we handle `spark.driver.extraJavaOptions` here?

Reply:
Forgot to add it. Will resolve this